Sprint-1/exercises/count.js

```diff
@@ -4,3 +4,6 @@ count = count + 1;
 
 // Line 1 is a variable declaration, creating the count variable with an initial value of 0
 // Describe what line 3 is doing, in particular focus on what = is doing
+
+
+// Line 3 is reassigning the value of the variable "count", to the value of "count + 1"
```

Comment:
There is a shorter ***programming term*** to describe operations like `count = count + 1` or `count++`. May I suggest feeding the code to ChatGPT to see how else the code can be described? From time to time you may learn new terminology or more concise ways to describe code.

Reply:
Yes, I am familiar with the ++ operator as well as the --.